node/bft/src/primary.rs

```diff
@@ -1538,7 +1541,7 @@ impl<N: Network> Primary<N> {
             // Ensure that the previous certificate was created at least `MIN_BATCH_DELAY_IN_MS` seconds ago.
```

Comment:
nit: This may have been missed 
```suggestion
            // Ensure that the previous certificate was created at least `MIN_BATCH_DELAY_IN_SECS` seconds ago.
```